scripts/script.js

```diff
@@ -0,0 +1,19 @@
+
+// Funktion för att stänga andra
+
+function toggle() {
+  var activeSections = document.getElementsByClassName("active");
```

Comment:
If possible - try use `const` or `let` before using `var`!
```suggestion
  const activeSections = document.getElementsByClassName("active");
```